src/DynamoCore/Models/NodeModel.cs

```diff
@@ -993,24 +993,12 @@ public void NotifyAstBuildBroken(string p)
 
         #region Port Management
 
-        internal int GetPortIndexAndType(PortModel portModel, out PortType portType)
+        internal int GetPortModelIndex(PortModel portModel)
```

Comment:
As `PortType` is a property in `PortModel`, here it is no need to return `portType`.